.buildkite/windows.ps1

```diff
@@ -0,0 +1,96 @@
+############################################################################
+# Windows build script for Buildkite
+#
+# Requires the Windows 10 VM to be set up with the compiler,
+# and other tools.
+#
+############################################################################
+
+Set-PSDebug -Trace 1
+
+Write-Output "--- Prepare work directory"
+
+# Avoid long paths on Windows
+$env:STACK_ROOT="D:\s"
+$env:STACK_WORK=".w"
+$env:WORK_DIR="D:\w"
+# Override the temp directory to avoid sed escaping issues
+# See https://github.com/haskell/cabal/issues/5386
+$env:TMP="D:\\tmp"
+
+# Store the original checkout directory
+$env:CHECKOUT_PATH=(Get-Item -Path ".\").FullName
+
+# Temp directory needs to exist
+New-Item -ItemType Directory -Force -Path $env:TMP
+
+$env:PATH="$env:PATH;D:\ghc\ghc-8.2.2\bin;D:\stack;$Env:Programfiles\7-Zip;D:\w"
+
+# Clear out work directory, copy source tree there, then cd into it.
+rd -r -fo $env:WORK_DIR
+mkdir $env:WORK_DIR
+copy-item $env:CHECKOUT_PATH\* $env:WORK_DIR -force -recurse
+cd $env:WORK_DIR
+
+# Pre-built rocksdb for Haskell
+git.exe clone https://github.com/facebook/rocksdb.git --branch v4.13.5
```

Comment:
Do we really need this if we use the pre-built one? If so, we should probably leave a comment here why we need the source checkout as well.

Reply:
Right, fixed the comment. It does need both. I tried removing the git clone before.